charts/latest-v2/azuredisk-csi-driver/templates/csi-azuredisk-scheduler-extender.yaml

```diff
@@ -24,7 +24,7 @@ data:
       filterVerb: "filter"
       prioritizeVerb: "prioritize"
       weight: 1
-      nodeCacheCapable: false
```

Comment:
We need to update [deploy/latest-v2/csi-azuredisk-scheduler-extender.yaml](https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/main_v2/deploy/latest-v2/csi-azuredisk-scheduler-extender.yaml#L26) as well.